npm/src/commands/search.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import {getGHLibsAndFlowVersions, filterDefs, formatDefTable}
+  from "../lib/libDef.js";
+import type {LibDefWithFlow} from "../lib/libDef.js";
+import {child_process, path} from '../lib/node'
+
+const P = Promise;
+
+export const name = "search";
+export const description =
+  "Performs a simple search (by name) of available libdefs";
+
+function findFlowVersion(): string {
+  const nodeModulesBin = path.resolve('node_modules', '.bin')
+  const _path = `${nodeModulesBin}:${process.env.PATH}`
+
+  function fail() {
+    console.error('Couldn\'t identify your flow version. Please make sure ' +
+    'flow is installed and in your PATH, or flow-bin is installed ' +
+    'in your project.')
+    process.exit(-1)
```

Comment:
Probably better to throw an `Error` than to `process.exit()`. That way anyone who potentially ends up using this elsewhere could, say, wrap the error message with some context (etc).